src/classes/groundplaces.ts

```diff
@@ -1,20 +1,29 @@
+import { Storage } from '../classes/storage';
 import {
   AutoComplete,
-  StopGroup,
-  StopCluster,
+  SegmentProviderStop,
+  StopGroupInfos,
+  StopClusterInfos,
   StopGroupGpuid,
   StopClusterGpuid,
   StopGroupProperties,
   StopClusterProperties,
   AutoCompleteFilters,
   GroundPlacesDiff,
   GroundPlaceType,
+  GroundPlacesObject,
 } from '../types';
 
 /**
  * @description GroundPlaces business logic.
  */
 export class GroundPlaces {
+  private readonly storageService: Storage;
+
+  constructor(groundPlacesFile: string) {
```

Comment:
If I understand, you gonna pass the json file through the constructor ? 
It's weird... what do you think @Theo-Le-Donne @Simobain ?

Reply:
I will be able to create a file initialization method, this is not present in specs, so as you want

Reply:
Don't really know what is the best way to do that.
What I know it's that it will load the s3 file directly in majority cases.
and a file uploaded through the studio sometimes

Reply:
Ok, I've create an `init` method for **GroundPlaces** class, which take as `parameter` the file where are the `ground places` If there is no file that is passed to this method, this will `init` the class with the file retrieved from S3

Reply:
i'd say you just move on, the init will obviously change when you implement the S3 methods, better not spend too much time. You already have enough abstraction to change only the file init part without changing anything else in another PR.